src/presenters/teaser-presenter.js

```diff
@@ -84,45 +87,56 @@ const TeaserPresenter = class TeaserPresenter {
 
 	//returns tag to be displayed
 	get displayTag () {
-		// Use Primary Tag is Primary Brand Tag the same as stream
-		if (this.data.streamProperties &&
-			this.data.streamProperties.idV1 &&
-			this.data.primaryBrandTag &&
-			this.data.streamProperties.idV1 === this.data.primaryBrandTag.idV1) {
-			return this.data.primaryTag || null;
-		}
-		// Use Author Tag if Opinion & Branded unless same as stream
-		if (brandAuthorDouble(this.data) === true &&
-			(!this.data.streamProperties ||
-			(this.data.streamProperties &&
-			this.data.streamProperties.idV1 !== this.data.authorTags[0].idV1 ))) {
-			return this.data.authorTags[0];
-		}
-		return this.data.primaryBrandTag || this.data.teaserTag || null;
+		//use package title as display tag if article belongs to package
+		let packageArticle = this.data.containedIn
+		if (packageArticle && packageArticle[0].title) {
```

Comment:
Could this be `packageArticle && packageArticle[0] && packageArticle[0].title` just in case